airflow/providers/airbyte/provider.yaml

```diff
@@ -50,6 +50,7 @@ versions:
 dependencies:
   - apache-airflow>=2.7.0
   - apache-airflow-providers-http
+  - airbyte-api>=0.51.0
```

Comment:
Do we still need http provider as dependency if we move to sdk?

Reply:
No, nice catch!